src/Microsoft.ApplicationInsights.AspNetCore/TelemetryInitializers/AzureWebAppRoleEnvironmentTelemetryInitializer.cs

```diff
@@ -11,7 +10,7 @@
     /// <summary>
     /// A telemetry initializer that will gather Azure Web App Role Environment context information.
     /// </summary>
-    public class AzureWebAppRoleEnvironmentTelemetryInitializer : TelemetryInitializerBase
+    internal class AzureWebAppRoleEnvironmentTelemetryInitializer : TelemetryInitializerBase
```

Comment:
I've noticed a lot of things being marked as internal (and the friend assembly reference for tests), any particular reason why?